runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -67,7 +59,7 @@ static const char* getTmpDir(void) {
 }
 
 // Check what version of slurm is on the system
-static sbatchVersion determineSlurmVersion(void) {
+static int checkSlurmVersion(void) {
```

Comment:
Please add a comment to say this function returns 1 on failure (and prints an error) and 0 on success

Reply:
Done.